bot/code_review_bot/workflow.py

```diff
@@ -322,6 +335,9 @@ def _in_group(dep_id):
                     continue
                 artifacts = task.load_artifacts(self.queue_service)
                 if artifacts is not None:
+                    if docupload_artifact_path in artifacts:
+                        link_to_doc = artifacts[docupload_artifact_path].decode("UTF-8")
```

Comment:
This code is very specific to docs, so it doesn't belong to the generic `workflow.py`, but it should be in the DocUploadTask class.

You can add a function such as `build_link` in bot/code_review_bot/tasks/base.py which returns `""` by default, and override it in DocUploadTask to return the link to the docs.
Then, in workflow.py you'd simply call `task.build_link`.